.github/workflows/pr-validation.yml

```
@@ -0,0 +1,132 @@
+  issues: write
+
+jobs:
+  validate-pr:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Check for Issue Reference
+        id: check_issue
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2086:info:7:30: Double quote to prevent globbing and word splitting

(shellcheck)